blockstack-sdk/build.gradle

```diff
@@ -79,14 +79,14 @@ apply plugin: 'kotlin-android-extensions'
 
 
 task downloadBlockstackJs(type: Download) {
-    src 'https://github.com/blockstack/blockstack.js/archive/v19.1.0.zip'
-    dest new File(buildDir, 'blockstack.js.zip')
+    src 'https://unpkg.com/blockstack@19.2.0-beta.2/dist/blockstack.js'
```

Comment:
We can now update this to: https://unpkg.com/blockstack@19.2.1/dist/blockstack.js